8.1.2/startup/initAppInsights.js

```diff
@@ -0,0 +1,8 @@
+try {
+    // Initialize the Application Insights runtime, using the
+    // instrumentation key provided by the environment.
+    require("applicationinsights").setup().start();
```

Comment:
Can the SDK take any kind of configuration in these calls? If the user is using the SDK, I think *their* call to it would be the second call (assuming --require would run before anything in the app); if they have done any kind of configuration, that should take precedence.